### ISSUE TITLE
Page scrolls up when I press shift-tab in nested list

### ISSUE THREAD

**Issue by mathias-ewald** [anyproto/anytype-ts#775]
### Have you read a contributing guide?

- [X] I have read CONTRIBUTING.md
- [X] I have searched the existing issues and didn't find any that were similar
- [X] I have considered creating a pull request with fixes instead of a bug report and want to proceed

### Current Behavior

When I create a nested unordered list, press enter for a new nested bullet and then press shift+tab to move a level up in the nesting, the page instantly "jump" (scrolls) up. The desired effect of moving a level up in the nesting takes effect, though.

### Expected Behavior

Page stays where it is.

### Steps To Reproduce

1. Create a lengthy document
1. Create unordered list
2. Create nested unordered list
3. Press enter after nested list item
4. Press shift-tab

### Environment

```markdown
- OS: MacOS Sonoma
- Version: 0.41.0
```

**Comment by ra3orblade** [comment]
Could you make a video, please? I can't reproduce this.

**Comment by mathias-ewald** [comment]
Sure, I attached it to the original post.

**Comment by mathias-ewald** [comment]
@ra3orblade Have you been able to reproduce now?

**Comment by ra3orblade** [comment]
> @ra3orblade Have you been able to reproduce now?

Not really, trying the same way that you describe, but works fine. I will pass this to our QA team.

**Comment by ra3orblade** [comment]
Closing as completed, feel free to re-open if it reproduces in latest versions.
